core/src/smartcontracts/wasm.rs

```diff
@@ -441,6 +475,48 @@ impl<'wrld> Runtime<'wrld> {
         self.execute_with_state(account_id, bytes, state)
     }
 
+    /// TODO
+    ///
+    /// # Errors
+    /// TODO
```

Comment:
will you add the description before merging?

Reply:
Oh, completely forgot about this, thanks